### PR TITLE
FI-1229 updated dependences

### PR DESCRIPTION
**Pull request by narkevicius**
### Description
Fixed `npm audit` issues. Updated `jest → 25.0.0`. Babel is gone.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,8489 @@
+{
```
Comment (twhid) [inline package-lock.json]: Please remove this file.

File: package.json [hunk 2]
```diff
@@ -35,11 +33,7 @@
         "intl-messageformat-parser": "^1.3.0"
     },
     "devDependencies": {
-        "babel-cli": "^6.26.0",
-        "babel-core": "^6.26.0",
-        "babel-eslint": "^8.0.1",
-        "babel-jest": "^21.2.0",
-        "babel-preset-env": "^1.6.1",
+        "babel-eslint": "^10.0.3",
```
Comment (twhid) [inline package.json]: Do we still need `babel-eslint` now that we've ditched babel? Or does `eslint-config-1stdibs` require it?

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,5 @@
 node_modules
 .vscode
-dist
\ No newline at end of file
+dist
```
Comment (kostasmanionis) [inline .gitignore]: Don't need this anymore?

File: package.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "icu-pseudo-translate",
-    "version": "2.0.0",
+    "version": "3.0.0",
```
Comment (kostasmanionis) [inline package.json]: Remove this, we'll change it when publishing

File: package.json (hunk 2 above)
Comment (narkevicius) [reply package.json]: Yes, it still requires it.

File: .gitignore (hunk 3 above)
Comment (narkevicius) [reply .gitignore]: No more.

**Pull request merged**